packages/map/src/components/EditorPanel/components/EditorPanel.tsx

```diff
@@ -374,6 +374,15 @@ const EditorPanel = () => {
           }
         })
         break
+      case 'legendGroupBy':
```

Comment:
In the future we want to move all of these switch statements back to where handleEditorChange is called to reduce the length here. Instead of:

```.js
  onChange={event => {
    handleEditorChanges('legendGroupBy', event.target.value)
  }}
  
  // can you use lodash and update the config directly:
    onChange={event => {
    const _newConfig = _.cloneDee(config)
    _newConfig.legend.subStyle = event.target.value
    setConfig(_newConfig)
  }}
  ```